test/somnium/test/congomongo.clj

```diff
@@ -223,7 +223,7 @@
           opts (.getMongoOptions m)]
       (testing "make-connection parses options from URI"
         (is (= 123 (.getConnectionsPerHost opts)))
-        (is (= WriteConcern/ACKNOWLEDGED (.getWriteConcern opts))))
+        (is (= WriteConcern/W1 (.getWriteConcern opts))))
```

Comment:
Looks like in `3.2` they changed `ACKNOWLEDGED` to be "everything defined by the server" and introduced `W1` to be what used to be `ACKNOWLEDGED`.